src/metatensor/models/utils/compute_loss.py

```diff
@@ -194,11 +194,11 @@ def _displacement_gradients_to_block(gradients_list):
 
     components = [
         Labels(
-            names=["cell_vector"],
+            names=["cell_abc"],
             values=torch.tensor([[0], [1], [2]]),
         ),
         Labels(
-            names=["coordinate"],
```

Comment:
Maybe also direction. I think we do it also for the position gradients in rascaline?

Reply:
I think this naming convention is something we should discuss tomorrow in ml-devel meeting, to make sure everyone uses the same names for everything in the lab.